webgui-new/src/enums/search-enum.ts

```diff
@@ -0,0 +1,62 @@
+export enum SearchMethods {
+  EXPRESSION = 'Search by expression',
+  FILE_REGEX = 'File name filter regex',
+  PATH_REGEX = 'Path filter regex',
+}
```

Comment:
Using the English labels as the enum values is not really a nice solution here. Currently only the English language is supported and we have no near future plans for supporting other languages, but it would have been still a nicer solution to place these into a real translation file and use [react-i18next](https://react.i18next.com/) for example for internationalization.

Reply:
@mdeme01 Created #607 to address this refactoring issue.